src/shared/forms/controllers/EmailInputController.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useForm, ErrorOption } from 'react-hook-form'
+
+import { EmailInputController } from 'shared/forms/controllers/EmailInputController'
+import { fireEvent, render, screen, waitFor } from 'tests/utils'
+
+type EmailForm = {
+  email: string
+}
+
+describe('<EmailInputController />', () => {
+  it('should show error when form input is invalid', () => {
+    renderEmailInputController({
+      error: { type: 'custom', message: 'error' },
+    })
+
+    expect(screen.getByText('error')).toBeTruthy()
+  })
+
```

Comment:
Ajouter le cas inverse
`it('should not show any error when form input is valid', () => {`